tests/runtime/inputs/AssertFail.hip

```diff
@@ -0,0 +1,10 @@
+// Regression test for https://github.com/CHIP-SPV/hipstar/issues/387.
+#include <hip/hip_runtime.h>
+
+__global__ void k() { __assert_fail("expression", "file", 123, "function"); }
```

Comment:
Can you add a test for the regular assert() macro that delegates to this then?

Reply:
Done.